app/_hub/kong-inc/ldap-auth-advanced/index.md

```diff
@@ -25,12 +26,10 @@ kong_version_compatibility:
         - 1.5.x
         - 1.3-x
         - 0.36-x
-        - 0.35-x
-        - 0.34-x
 
 params:
   name: ldap-auth-advanced
-  api_id: true
+  api_id: false
```

Comment:
It's better to just remove this line. They technically perform the same function, but simply removing it will help avoid any questions in the future (and one less piece of metadata to keep building).

Same goes for any other plugins/versions, and sorry if I missed this in previous reviews - you don't need to go back and remove, but just take it out going forward.